packages/react/roving-focus/src/RovingFocusGroup.tsx

```diff
@@ -173,4 +178,6 @@ export {
   Root,
   //
   useRovingFocus,
+  //
+  filterVisibleItems,
```

Comment:
we could not export this, but if so, we need to copy this logic to the Menu (considering it's just a one liner, it should be fine, if you prefer it that way)